test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/TestResources.cs

```diff
@@ -10,11 +10,15 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding
     // internal properties.
     public static class TestResources
     {
-        public static string DisplayAttribute_Description { get; } = Resources.DisplayAttribute_Description;
+        public static string Type_Three_Name => "type three name " + CultureInfo.CurrentCulture;
+        public static string Type_Three_Description => "type three description " + CultureInfo.CurrentCulture;
+        public static string Type_Three_Prompt => "type three prompt " + CultureInfo.CurrentCulture;
```

Comment:
Please double-check that `[ReplaceCulture]` or `using (new CultureReplacer(...))` is used in all tests referencing the `CurrentCulture` properties from this class.